### DISCUSSION TITLE
Copilot is not working, stucks with Synthesizing solutions

### DISCUSSION THREAD

**Discussion by daubac402** [community/community#6867] [Copilot Conversations] [answered]
For me, Github Copilot doesn't give any suggest.
When I `Ctrl` + `Shift` + `P` and select `Open Github Copilot`
It stucks with:
```
Synthesizing 0/10 solutions
```

Anyone else?

**Comment by davecheney** [comment, marked as answer]
Are you behind a firewall, VPN, or other device that may be intercepting your HTTPS connections?

**Reply by daubac402** [reply]
Oh, yes. I am behind a Fortigate firewall and it has a custom CA certificate for all HTTPS traffic.

**Reply by davecheney** [reply]
I'm sorry about that. There seems to be a common pattern that these appliances which are man-in-the-middling your SSL traffic tend to break the connection rather than snoop on it. There isn't really much I can do to change that, but I'm happy to have a conversation with the administrators of those devices if they would like to set up an exception to not intercept your copilot traffic.

**Reply by daubac402** [reply]
Thanks, I got it. Our administrator checked the log and then It's OK for now 👍

**Reply by PatoLocos** [reply]
The only version I can run is v1.1.1909, other don't seem to work.
And when I run it I get "Synthesizing 0/10 solutions" message. 
I have 2 windows 10 machine one for work and one private in both I have the same problem.

**Reply by davecheney** [reply]
Any version before 1.6 will no longer work. You must upgrade.

**Reply by PatoLocos** [reply]
I have done it too,   then I get stuck with this one 
https://stackoverflow.com/questions/68253302/github-copilot-commands-not-working-and-showing-error
Same in both computers.

**Reply by davecheney** [reply]
@PatoLocos I cannot see any connections from your token to our servers. It looks like your connections are being blocked. Sorry, that is all the assistance I can provide.

**Reply by PatoLocos** [reply]
Thanks Dave, that confirms it has to be some kind firewall policy on my PCs, I will try to solved and publish back a solution.

**Reply by mohamadhussien** [reply]
I had the same issue, solved it by accepting the pop up notification in VSCode that ask for telemetry and restarting VSCode.

**Reply by Victorvikson1996** [reply]
How do i do that

**Reply by daubac402** [reply]
Please check your connection to **copilot-proxy.trafficmanager.net tcp/443**
After I allow connection to this one, It works :)

**Reply by dappgbut** [reply]
How do you check the connection to that? Using command prompt?
